SFUsat/sfu_cmds.h

```diff
@@ -98,18 +98,16 @@
  */
 #define CMD_DATA_MAX_SIZE (14)
 
```

Comment:
Could you add a blurb here similar to the comments previously explaining the magic numbers?

E.g. like
```
/**
 * Magic numbers to identify a task.
 * - Definition order does not matter.
 * - Exact value does not matter.
 * - Must fit within a uint8_t (0x00 to 0xFF).
 */
#define TASK_MAIN           (0x00)
...
```